### DISCUSSION TITLE
pycaret using the Xe Graphics gpu and not the NVIDIA GeForce RTX 4050

### DISCUSSION THREAD

**Discussion by dkinneyBU** [pycaret/pycaret#4065] [Q&A]
with use_gpu=True, pycaret is defaulting to the generic Xe Graphics gpu. Is there a way to tell pycaret to use the NVIDIA gpu?

thank you!

**Comment by celestinoxp** [comment]
Can you give more details? Algortihm in use, dataset size, SO, etc...

**Reply by BradKML** [reply]
For clarity, either for laptop for other desktop with onboard graphics, PyCaret will choose to use Intel Graphics over NVidia with the newest version of PyCaret `--pre`, and through using the default classifier/regressor with `use_gpu=True`.
```
[LightGBM] [Info] Using GPU Device: Intel(R) UHD Graphics, Vendor: Intel(R) Corporation  
[LightGBM] [Info] Compiling OpenCL Kernel with 16 bins...
[LightGBM] [Info] GPU programs have been built
[LightGBM] [Warning] GPU acceleration is disabled because no non-trivial dense features can be found
```

**Comment by celestinoxp** [comment]
Can you get Nvidia Control Panel? I think there you can force specific apps to use the Nvidia graphics card

**Reply by BradKML** [reply]
That is true, in that case should we be pointing just the IDE? Or are things like the Python environment or command line need to also prioritize RTX over the Intel default?
